test/test_wait.py

```diff
@@ -0,0 +1,90 @@
+from opensquirrel import Circuit, CircuitBuilder
+from opensquirrel.ir import Bit
+
+
+def test_wait() -> None:
```

Comment:
I would be interested in a test utilizing a multiqubit gate, barriers and waits to see if the logic flows correctly.

Reply:
Any suggestions?

Reply:
```
builder = CircuitBuilder(3)
  builder.H(0)
  builder.H(1)
  builder.wait(0, 1)
  builder.barrier(1)
  builder.wait(3,1)
  builder.barrier(0)
  builder.CNOT(0, 1)
  builder.wait(0, 3)
```